### ISSUE TITLE
base_url should include port?

### ISSUE THREAD

**Issue by czarop** [LukeMathWalker/zero-to-production#270]
In the code around generating a subscription confirmation link for the email, the link address is stated as being '{base_url}/subscriptions/confimation...' (base_url being taken from base.yaml for the local version of the app). 
In base.yaml, base_url is "http://127.0.0.1" - I _believe_ this should be 'http://127.0.0.1:8000' , otherwise the confirmation link generated does not work locally.

**Comment by ajay-vishnu** [comment]
~Same doubt. Did you figure it out?~

EDIT:
It's misguiding because of the new return type of `Url::parse()` method. You just have to continue reading the book.

**Comment by czarop** [comment]
Yes - did figure that out eventually! should've posted here sorry!! I'll close this issue.
